HealthcareSystem/UserService/Model/MaliciousAction.cs

```diff
@@ -19,7 +20,7 @@ public MaliciousAction(MaliciousActionType type, DateTime timeStamp)
 
         private void Validate()
         {
-            throw new NotImplementedException();
+            if (!Type.Equals(MaliciousActionType.AppointmentCancellation)) throw new ValidationException("Type of malicious action must be appointment cancellation.");
```

Comment:
Eh, I'd delete this, it's not like it can be anything else. And if we add something else we'd have to change this, so it's just a pain.

Reply:
Should I delete the method: Validate()?